exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/SQLServerDialect.kt

```diff
@@ -149,7 +149,11 @@ open class SQLServerDialect : VendorDialect(dialectName, SQLServerDataTypeProvid
     override val supportsSequenceAsGeneratedKeys: Boolean = false
     override val supportsOnlyIdentifiersInGeneratedKeys: Boolean = true
 
-    override fun isAllowedAsColumnDefault(e: Expression<*>): Boolean = true
+    override fun isAllowedAsColumnDefault(e: Expression<*>): Boolean {
+        val nonAcceptableDefaults = arrayOf("DEFAULT")
```

Comment:
I would advise moving that as a `val` on dialect level or replace it with simple `!= "DEFAULT"`

Reply:
Agreed. Moved it up.